src/components/InputBox.jsx

```diff
@@ -0,0 +1,15 @@
+import React from  "react";
+
+const InputBox = () => {
+    return (
+        <form>
+  <label>
+    Name:
+    <input type="text" name="name" />
+  </label>
+  <input type="submit" value="Submit" />
+</form>
+    );
+}
```

Comment:
Great work here, just need to fix up the formatting
Are you using VSCode? If you are , install the Prettier code formatter addon and then run it on this file